### DISCUSSION TITLE
Renaming AI-on-the-edge-device

### DISCUSSION THREAD

**Discussion by JAudi23** [jomjol/AI-on-the-edge-device#2473] [General] [answered]
Currently my device that has been working great for a few months has my original generic hostname of "esp32-cam-1" as I was playing around with ESPHome, prior to installing this.

Is there a simple way to rename the host to a new name that I can make more relative to it's use case?

Sorry if this has been answered before, or if it's an obvious solution.

Thanks,

**Comment by friedpa** [comment, marked as answer]
Settings -> Configuration -> Expert View -> Hostname

**Reply by JAudi23** [reply]
Well that was obvious.   Not sure how I missed it, thank you.

**Reply by skupu** [reply]
In my case it is only available in direct config.ini edit window

**Reply by caco3** [reply]
Did you enable the expert mode?

https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-hostname
